packages/cli-lib/github.js

```diff
@@ -182,26 +182,43 @@ async function downloadGitHubRepoContents(
         `Downloading content piece: ${contentPiecePath} from ${download_url} to ${downloadPath}`
       );
 
-      return fetchGitHubRepoContentFromDownloadUrl(downloadPath, download_url, {
-        headers: { ...DEFAULT_USER_AGENT_HEADERS, ...GITHUB_AUTH_HEADERS },
-      });
+      if (contentPiece.type === 'dir') {
+        const contentPath = contentPiece.path;
```

Comment:
Nit, it looks like you've already destructured this `path` field [here](https://github.com/HubSpot/hubspot-cli/pull/824/files#diff-e33aaeaafb6096b335e754b56e4dbfab3afa2f136b6ad3f4e448ced2493e8fc9R168).

You could also destrcuture `type` for consistency